pyxnat/core/derivatives/donsurf.py

```diff
@@ -0,0 +1,8 @@
+XNAT_RESOURCE_NAME = 'DONSURF'
+
+
+def aparc(self):
+    """Returns cortical features as estimated by `DONSURF`."""
+
+    from .freesurfer import aparc
+    return aparc(self)
```

Comment:
That's a bit mind-blowing, consider the following:
```suggestion
    from .freesurfer import fs_aparc
    return fs_aparc(self)
```

Reply:
9d6cec1 fixes this.